From a0ee7592c739da4899dca0c9d10a9cd03bcc4e4a Mon Sep 17 00:00:00 2001
From: Gustavo Branco <guster32@gmail.com>
Date: Sun, 8 Aug 2021 00:43:33 -0700
Subject: [PATCH] No gbm_bo_map
Upstream-Status: Inappropriate [oe specific, temp hack]

---
 cube-tex.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cube-tex.c b/cube-tex.c
index c8f24c0..a62d03a 100644
--- a/cube-tex.c
+++ b/cube-tex.c
@@ -228,7 +228,7 @@ static int get_fd_rgba(uint32_t *pstride, uint64_t *modifier)
 	/* NOTE: do not actually use GBM_BO_USE_WRITE since that gets us a dumb buffer: */
 	bo = gbm_bo_create(gl.gbm->dev, texw, texh, GBM_FORMAT_ABGR8888, GBM_BO_USE_LINEAR);

-	map = gbm_bo_map(bo, 0, 0, texw, texh, GBM_BO_TRANSFER_WRITE, &stride, &map_data);
+	//map = gbm_bo_map(bo, 0, 0, texw, texh, GBM_BO_TRANSFER_WRITE, &stride, &map_data);

 	for (uint32_t i = 0; i < texh; i++) {
 		memcpy(&map[stride * i], &src[texw * 4 * i], texw * 4);
@@ -263,7 +263,7 @@ static int get_fd_y(uint32_t *pstride, uint64_t *modifier)
 	/* NOTE: do not actually use GBM_BO_USE_WRITE since that gets us a dumb buffer: */
 	bo = gbm_bo_create(gl.gbm->dev, texw, texh, GBM_FORMAT_R8, GBM_BO_USE_LINEAR);

-	map = gbm_bo_map(bo, 0, 0, texw, texh, GBM_BO_TRANSFER_WRITE, &stride, &map_data);
+	// map = gbm_bo_map(bo, 0, 0, texw, texh, GBM_BO_TRANSFER_WRITE, &stride, &map_data);

 	for (uint32_t i = 0; i < texh; i++) {
 		memcpy(&map[stride * i], &src[texw * i], texw);
@@ -298,7 +298,7 @@ static int get_fd_uv(uint32_t *pstride, uint64_t *modifier)
 	/* NOTE: do not actually use GBM_BO_USE_WRITE since that gets us a dumb buffer: */
 	bo = gbm_bo_create(gl.gbm->dev, texw/2, texh/2, GBM_FORMAT_GR88, GBM_BO_USE_LINEAR);

-	map = gbm_bo_map(bo, 0, 0, texw/2, texh/2, GBM_BO_TRANSFER_WRITE, &stride, &map_data);
+	// map = gbm_bo_map(bo, 0, 0, texw/2, texh/2, GBM_BO_TRANSFER_WRITE, &stride, &map_data);

 	for (uint32_t i = 0; i < texh/2; i++) {
 		memcpy(&map[stride * i], &src[texw * i], texw);
