diff --git a/tools/Makefile b/tools/Makefile
index 2fc4a583d4..ac30086021 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -114,8 +114,8 @@ fit_check_sign-objs   := $(dumpimage-mkimage-objs) fit_check_sign.o
 
 # Build a libfdt Python module if swig is available
 # Use 'sudo apt-get install swig libpython-dev' to enable this
-hostprogs-y += \
-	$(if $(shell which swig 2> /dev/null),_libfdt.so)
+#hostprogs-y += \
+#	$(if $(shell which swig 2> /dev/null),_libfdt.so)
 _libfdt.so-sharedobjs += $(LIBFDT_OBJS)
 libfdt:
 
